ui/pages/confirmations/send/send-content/add-recipient/domain-input.component.js

```diff
@@ -92,9 +92,12 @@ export default class DomainInput extends Component {
         !isBurnAddress(input) &&
         isValidHexAddress(input, { mixedCaseUseChecksum: true })
       ) {
-        onValidAddressTyped(addHexPrefix(input));
+        const hexInput = addHexPrefix(input);
+        onChange(hexInput);
+        onValidAddressTyped(addHexPrefix(hexInput));
```

Comment:
Why is `input` wrapped on `addHexPrefix` twice?

Reply:
Good catch, it is a redudant addPrefix, let me remove the last one.